src/extensions/spikeamplitudes/SpikeAmplitudesView/SpikeAmplitudesTimeWidget.tsx

```diff
@@ -29,17 +29,26 @@ const SpikeAmplitudesTimeWidget: FunctionComponent<Props> = ({ spikeAmplitudesDa
 
     useEffect(() => {
         const panels: SpikeAmplitudesPanel[] = []
+        const allMins: number[] = []
+        const allMaxs: number[] = []
+
         unitIds.forEach(unitId => {
             const p = new SpikeAmplitudesPanel({spikeAmplitudesData, recording, sorting, unitId, hither})
+            const amps = spikeAmplitudesData.getSpikeAmplitudes(unitId)
```

Comment:
This stuff all goes along with new method for determining the auto scale factor for spike amplitudes view